src/instruments/src/RMP/RadioManagementPanel.mjs

```diff
@@ -0,0 +1,80 @@
+import { VeryHighFrequency } from './RadioFrequencyMode.mjs';
+import { HighFrequency } from './RadioFrequencyMode.mjs';
+import { AcceleratedKnob } from "./AcceleratedKnob.mjs";
+import { useInteractionEvent } from '../util.mjs';
+
+
+const vhf1 = new VeryHighFrequency(1);
+const vhf2 = new VeryHighFrequency(2);
+const vhf3 = new VeryHighFrequency(3);
+const hf1 = new HighFrequency(1);
+const hf2 = new HighFrequency(2);
+
+class RadioManagementPanel {
+    constructor(side) {
+        this.side = side;
+        this.mode = side === 'L' ? vhf1 : vhf2;
+        this.innerKnob = new AcceleratedKnob();
+        this.outerKnob = new AcceleratedKnob();
+        this.innerKnob.updateValue = (s) => this.mode.changeStandbyDecimalValue(s);
+        this.outerKnob.updateValue = (s) => this.mode.changeStandbyIntegerValue(s);
+    }
+
+    frequencyKnobTurned(knob, direction) {
+        if (direction === 'anticlockwise') knob.decrease();
+        else if (direction === 'clockwise') knob.increase();
+        if (this.requestRender) this.requestRender();
+    }
+
+    transferButtonPressed() {
+        this.mode.transfer()
+        if (this.requestRender) this.requestRender();
+    }
+
+    modeButtonPressed(mode) {
+        this.mode = mode;
+        if (this.requestRender) this.requestRender();
+    }
+
+    registerCallbacks(requestRender) {
+        this.requestRender = requestRender;
```

Comment:
For readability, I would just avoid thinking about renders here at all, also from a naming scheme. How about calling this `registerCallbacks(changeListener)`? Then it's clearer that this just a class that observes events and simvars and notifies a callback, rather than something to do with React.